data/bootstrap_seed.yaml

```diff
@@ -0,0 +1,171 @@
+---
+webvirt: 192.168.1.1
+site_name: sitename
+maas_api_key: consumerkey:tokenkey:tokensecret
```

Comment:
No, API key could be configured during service initialization. UUID is part of the oauth stuff for MaaS as well I think. So this could be dropped from here and added to the MaaS deployment chart. I'll pull both out. I do think leaving the site_name in place is needed.